Sources/ImageFetcher/ImageConfiguration.swift

```diff
@@ -58,3 +68,12 @@ extension ImageConfiguration: Keyable {
         return keyValues.joined().md5
     }
 }
+
+extension ImageConfiguration: ExpressibleByStringLiteral {
```

Comment:
It's purely for the convenience of users. The idea is you could replace:

```swift
let config = ImageConfiguration(url: URL(string: "https://via.placeholder.com/150")!)
fetcher.load(config) { ... }
```

with:

```swift
fetcher.load("https://via.placeholder.com/150")
```

Thinking about it now, it would probably be more appropriate to add that conformance to `URL` rather than `ImageConfiguration`